man/tpm2_eventlog.1.md

```diff
@@ -0,0 +1,39 @@
+% tpm2_eventlog(1) tpm2-tools | General Commands Manual
+
+# NAME
+
+**tpm2_eventlog**(1) - Display tpm2 event log.
+
+# SYNOPSIS
+
+**tpm2_eventlog**  [*ARGUMENT*]
+
+# DESCRIPTION
+
+**tpm2_eventlog**(1) - Parse a binary TPM2 event log. The event log may be
+passed to the tool as the final positional parameter. If this parameter is
+omitted the tool will return an error. The format of this log documented in
+the "TCG PC Client Platform Firmware Profile Specification".
+
+# OPTIONS
+
+This tool takes no tool specific options.
+
+  * **ARGUMENT** The command line argument is the path to a binary TPM2
+    eventlog.
+
+## References
+
+[common options](common/options.md) collection of common options that provide
```

Comment:
This has -Q in it, ill fix up the quiet code and pull this in, you got it far enough

Reply:
Thanks.